testutil/sims/simulation_helpers.go

```diff
@@ -68,7 +68,7 @@ func SimulationOperations(app runtime.AppI, cdc codec.JSONCodec, config simtypes
 		}
 	}
 
-	simState.LegacyProposalContents = app.SimulationManager().GetProposalContents(simState) //nolint:staticcheck
+	simState.LegacyProposalContents = app.SimulationManager().GetProposalContents(simState) //nolint:staticcheck // this supports proposal contents, instead of the new proposal messages
```

Comment:
I would write is as support for gov v1beta1 proposals. However this somewhat okay too.

Reply:
yours is better.